release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2581,3 +2632,36 @@ link:https://access.redhat.com/solutions/6762051[{product-title} 4.9.23 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-24"]
+=== RHBA-2022:0798 - {product-title} 4.9.24 bug fix update
+
+Issued: 2022-03-16
+
+{product-title} release 4.9.24 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0798[RHBA-2022:0798] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:0794[RHBA-2022:0794] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6818041[{product-title} 4.9.24 container image list]
+
+[id="ocp-4-9-24-features"]
+==== Features
+
+This update contains changes from Kubernetes 1.22.5. More information can be found in the following changelog: link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#changelog-since-v1225[1.22.3].
+
+[id="ocp-4-9-24-removed-feature-azure-mint-mode"]
+==== Removed features
+
+Starting with {product-title} 4.9.24, support for using the Cloud Credential Operator (CCO) in mint mode on Microsoft Azure clusters has been removed from {product-title} 4.9. This change is due to the planned link:https://azure.microsoft.com/en-us/updates/update-your-apps-to-use-microsoft-graph-before-30-june-2022/[retirement of the Azure AD Graph API by Microsoft on 30 June 2022] and is being backported to all supported versions of {product-title} in z-stream updates. For more information, see xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-feature-azure-mint-mode[Support for minting credentials for Microsoft Azure removed].
+
+[id="ocp-4-9-24-bug-fixes"]
+==== Bug fixes
+
+* Previously, the console failed to upload an edit page for *Deployment*, *DeploymentConfig*, and *Serverless Service*. This update adds a check for a loading error, and the page now displays a *Not Found* message rather than failing to load. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])
+
+* Previously, a rewrite in the `kube-apiserver` caused a user facing API change. Users must explicitly specify either `ipFamilyPolicy:PreferDualStack` or `ipFamilyPolicy:RequireDualStack` for DualStack services to be `valid.Users`. With this update, a warning is emitted through the API that notifies users of the imminent API change. Now, users cannot create DualStack services without explicitly specifying `ipFamilyPolicy:PreferDualStack` or `ipFamilyPolicy:RequireDualStack`.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2045576[*BZ#2045576*])
```

Comment:
First sentence: change "user facing" to "user-facing."

Second sentence: change "emitted" to something less formal. Change "that notifies" to "notifying."